modules/oe_content_entity/modules/oe_content_entity_organisation/oe_content_entity_organisation.info.yml

```diff
@@ -7,3 +7,23 @@ core: 8.x
 dependencies:
   - oe_content:oe_content_entity
   - oe_media:oe_media
+
+config_devel:
+  install:
+    - core.base_field_override.oe_organisation.oe_stakeholder.name
+    - core.base_field_override.oe_organisation.oe_stakeholder.status
```

Comment:
Remove these two as we don't export base field overrides:
```
    - core.base_field_override.oe_organisation.oe_stakeholder.name
    - core.base_field_override.oe_organisation.oe_stakeholder.status
```

Reply:
done